src/pages/LoginSignupPage/LoginSignupPage.scss

```diff
@@ -0,0 +1,39 @@
+* {
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  box-sizing: border-box;
+}
+
+.loginSignupPage__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
```

Comment:
in Globals.scss we should store  height: 100vh; (the 100vh should be default height on every place in app)